PullRequestMonitor/Model/TfProject.cs

```diff
@@ -115,11 +98,14 @@ public void DoPullRequestRetrieval()
             try
             {
                 var getPRsTask = _tfsConnection.GetActivePullRequestsInProject(this);
+                var getCompletedPullRequestsTask = _tfsConnection.GetCompletedPullRequestsInProject(this);
 
                 getPRsTask.Wait();
+                getCompletedPullRequestsTask.Wait();
 
                 var activePullRequestIds = EnsureActivePrsArePresent(getPRsTask.Result);
                 RemoveStalePullRequests(activePullRequestIds);
+                EnsureCompletedPrsArePresent(getCompletedPullRequestsTask.Result);
```

Comment:
Would be great to do this only when needed if it doesn't complicate things too badly. I think the MVVM is okay in this area so hopefully not.